Add fastapi to the list of compose projects

vjanz opened the pull request.

vjanz commented:
What is happening here?

espoirMur commented on fastapi/Dockerfile:
```diff
@@ -0,0 +1,12 @@
+FROM python:3.8
```
can you use the official tiagolo fastapi docker file here? 
`tiangolo/uvicorn-gunicorn-fastapi`

vjanz replied:
Hey @espoirMur,

Sure, changed the docker image as requested to the pre-configured image of tiangolo. 

Cheers!

espoirMur reviewed (commented):
Can you check my comment on the dockerfile? 
Beside that the code looks good for me , thanks you for adding this

vjanz commented:
Updated and changed as requested. As per two other file changes, I'm not sure why it's being included in my PR, but if you would like I can create a new PR with the recent changes only.

Cheers!
